Dockerfile.template

```diff
@@ -67,12 +67,14 @@ RUN buildDeps=' \
 		libsqlite3-dev \
 		make \
 		patch \
+		libgmp-dev \
+		libc6-dev \
```

Comment:
Are you sure these are necessary?  I was able to install `sqlserver` successfully without them in my testing. :confused:

(If they are indeed necessary, this list should stay in sorted order, but I'm not entirely convinced they are -- maybe they're pulled in already by one of the above deps?)

Reply:
You might be correct. I haven't been able to figure out how to test this locally due to a variety of Docker errors I encountered. I did have to install these packages before attempting to add support for SQL server in order to build the TinyTDS gem. I will try removing these and seeing whether the CI build passes.